### PR TITLE
Use a consistent term: kwok and kwokctl

### LINKED ISSUE #340: Whether we need use a consistent term about kwok and kwokctl
### What would you like to be added?

Currently, we have 4 variants in the repo and website:

- KWOK
- kwok, kwokctl
- Kwok, Kwokctl
- `kwok`, `kwokctl`

Could we reduce those terms to 2 variants as follows:
- KWOK, when we describe it is a project, product, brand
- `kwok` and `kwokctl`, when we describe it is a component of KWOK or it is displayed on CLI or code snippet

### Why is this needed?

Consistency is a part of quality.

### PR DESCRIPTION
**Pull request by windsonsea**
#### What type of PR is this?

/kind documentation

#### What this PR does / why we need it:

Try to use a consistent expression about kwok and kwokctl:

- If a product name, use KWOK same as the logo
- If a component, use kwok and kwokctl (both in lower case)
- If a code snippet or UI is nearby, use `kwok` and `kwokctl` to describe (This may need to discuss further)

#### Does this PR introduce a user-facing change?

```release-note
NONE
```
Fixes #340

### CODE REVIEW

File: site/content/en/docs/user/install.md [hunk 1]
```diff
@@ -19,21 +19,21 @@ brew install kwok
 ### Variables preparation
 
 ``` bash
-# Kwok repository
+# kwok repository
```
Comment (wzshiming) [inline site/content/en/docs/user/install.md:22]:
```suggestion
# KWOK repository
```

File: site/content/en/docs/user/kwok-in-cluster.md [hunk 2]
```diff
@@ -11,7 +11,7 @@ This document walks you through how to deploy `kwok` in a Kubernetes cluster.
 ``` bash
 # Temporary directory
 KWOK_WORK_DIR=$(mktemp -d)
-# Kwok repository
+# kwok repository
```
Comment (wzshiming) [inline site/content/en/docs/user/kwok-in-cluster.md:14]:
```suggestion
# KWOK repository
```

File: site/content/en/menu/index.md [hunk 3]
```diff
@@ -7,12 +7,12 @@ headless: true
   - **Quick Start**
     - [All in One Image]({{< relref "/docs/user/all-in-one-image" >}})
     - [Install]({{< relref "/docs/user/install" >}})
-    - [Kwok in Cluster]({{< relref "/docs/user/kwok-in-cluster" >}})
-    - [Kwok in Local]({{< relref "/docs/user/kwok-in-local" >}})
-    - [Kwokctl Manage Clusters]({{< relref "/docs/user/kwokctl-manage-cluster" >}})
+    - [kwok in Cluster]({{< relref "/docs/user/kwok-in-cluster" >}})
```
Comment (wzshiming) [inline site/content/en/menu/index.md:10]: Add below in `site/assets/_custom.scss` and also wrapping it in '`'
``` scss
code {
    padding: 0 .25rem;
    background: var(--gray-200);
    border-radius: .25rem;
    font-size: .875em;
}
```

File: site/content/en/_index.md [hunk 4]
```diff
@@ -20,27 +20,27 @@ KWOK stands for Kubernetes WithOut Kubelet. So far, it provides two tools:
...
 ### Basic Usage
 
-- [Kwok Manage Nodes and Pods]({{< relref "/docs/user/kwok-manage-nodes-and-pods" >}}) - Kwok's basic management of Node and Pod operations
-- Kwok - maintain Nodes heartbeat and Pods status.
-    - [Kwok in Cluster]({{< relref "/docs/user/kwok-in-cluster" >}}) - Installing Kwok in a cluster
-    - [Kwok in Local]({{< relref "/docs/user/kwok-in-local" >}}) - Run Kwok in the local for a cluster
-- Kwokctl - cluster creation, etcd snapshot, etc.
-    - [Kwokctl Manage Clusters]({{< relref "/docs/user/kwokctl-manage-cluster" >}}) - Create/Delete a cluster in local where all nodes are managed by Kwok
-    - [Kwokctl Snapshot Cluster]({{< relref "/docs/user/kwokctl-snapshot" >}}) - Save/Restore the Etcd data of a cluster created by Kwokctl
+- [kwok Manages Nodes and Pods]({{< relref "/docs/user/kwok-manage-nodes-and-pods" >}}) - Basic operations of `kwok` to manage Nodes and Pods
```
Comment (wzshiming) [inline site/content/en/_index.md:29]:
```suggestion
- [`kwok` Manages Nodes and Pods]({{< relref "/docs/user/kwok-manage-nodes-and-pods" >}}) - Basic operations of `kwok` to manage Nodes and Pods
```

File: site/content/en/_index.md [hunk 5]
```diff
@@ -20,27 +20,27 @@ KWOK stands for Kubernetes WithOut Kubelet. So far, it provides two tools:
...
+- kwok - maintain Nodes heartbeat and Pods status.
```
Comment (wzshiming) [inline site/content/en/_index.md:30]:
```suggestion
- `kwok` - maintain Nodes heartbeat and Pods status.
```

File: site/content/en/_index.md [hunk 6]
```diff
@@ -20,27 +20,27 @@ KWOK stands for Kubernetes WithOut Kubelet. So far, it provides two tools:
...
+    - [kwok in Cluster]({{< relref "/docs/user/kwok-in-cluster" >}}) - Installing `kwok` in a cluster
```
Comment (wzshiming) [inline site/content/en/_index.md:31]:
```suggestion
    - [`kwok` in Cluster]({{< relref "/docs/user/kwok-in-cluster" >}}) - Installing `kwok` in a cluster
```

File: site/content/en/_index.md [hunk 7]
```diff
@@ -20,27 +20,27 @@ KWOK stands for Kubernetes WithOut Kubelet. So far, it provides two tools:
...
+    - [kwok in Local]({{< relref "/docs/user/kwok-in-local" >}}) - Run `kwok` in the local for a cluster
```
Comment (wzshiming) [inline site/content/en/_index.md:32]:
```suggestion
    - [`kwok` in Local]({{< relref "/docs/user/kwok-in-local" >}}) - Run `kwok` in the local for a cluster
```

File: site/content/en/_index.md [hunk 8]
```diff
@@ -20,27 +20,27 @@ KWOK stands for Kubernetes WithOut Kubelet. So far, it provides two tools:
...
+- kwokctl - cluster creation, etcd snapshot, etc.
```
Comment (wzshiming) [inline site/content/en/_index.md:33]:
```suggestion
- `kwokctl` - cluster creation, etcd snapshot, etc.
```

File: site/content/en/_index.md [hunk 9]
```diff
@@ -20,27 +20,27 @@ KWOK stands for Kubernetes WithOut Kubelet. So far, it provides two tools:
...
+    - [kwokctl Manages Clusters]({{< relref "/docs/user/kwokctl-manage-cluster" >}}) - Create/Delete a cluster in local where all nodes are managed by `kwok`
```
Comment (wzshiming) [inline site/content/en/_index.md:34]:
```suggestion
    - [`kwokctl` Manages Clusters]({{< relref "/docs/user/kwokctl-manage-cluster" >}}) - Create/Delete a cluster in local where all nodes are managed by `kwok`
```

File: site/content/en/_index.md [hunk 10]
```diff
@@ -20,27 +20,27 @@ KWOK stands for Kubernetes WithOut Kubelet. So far, it provides two tools:
...
+    - [kwokctl Snapshots Cluster]({{< relref "/docs/user/kwokctl-snapshot" >}}) - Save/Restore the Etcd data of a cluster created by `kwokctl`
```
Comment (wzshiming) [inline site/content/en/_index.md:35]:
```suggestion
    - [`kwokctl` Snapshots Cluster]({{< relref "/docs/user/kwokctl-snapshot" >}}) - Save/Restore the Etcd data of a cluster created by `kwokctl`
```

File: site/content/en/_index.md [hunk 11]
```diff
@@ -20,27 +20,27 @@ KWOK stands for Kubernetes WithOut Kubelet. So far, it provides two tools:
...
 
 ### Contributing
 
 - [Contributing]({{< relref "/docs/contributing/getting-started" >}}) - How to contribute to KWOK
 
-## Kwokctl Runtime and OS Support
+## kwokctl Runtime and OS Support
```
Comment (wzshiming) [inline site/content/en/_index.md:41]:
```suggestion
## `kwokctl` Runtime and OS Support
```

File: site/content/en/docs/user/kwok-manage-nodes-and-pods.md [hunk 12]
```diff
@@ -1,22 +1,22 @@
...
+## kwok with args `--manage-all-nodes=true`
 
-Kwok will be in charge of all nodes in the cluster and maintain their heartbeats to API Server. In this way, all nodes will behave like real nodes and stay in `Ready` state.
+kwok will be in charge of all nodes in the cluster and maintain their heartbeats to API Server. In this way, all nodes will behave like real nodes and stay in `Ready` state.
 
-## Kwok with args `--manage-nodes-with-annotation-selector=kwok.x-k8s.io/node=fake`
+## kwok with args `--manage-nodes-with-annotation-selector=kwok.x-k8s.io/node=fake`
 
-Kwok will be in charge of all Pods with annotation `kwok.x-k8s.io/node=fake`. If they carry an accurate `.spec.nodeName` filed, Kwok will ensure they stay in the `Running` state.
+kwok will be in charge of all Pods with annotation `kwok.x-k8s.io/node=fake`. If they carry an accurate `.spec.nodeName` filed, kwok will ensure they stay in the `Running` state.
```
Comment (wzshiming) [inline site/content/en/docs/user/kwok-manage-nodes-and-pods.md:15]:
```suggestion
`kwok` will be in charge of all Pods with annotation `kwok.x-k8s.io/node=fake`. If they carry an accurate `.spec.nodeName` field, `kwok` will ensure they stay in the `Running` state.
```

File: site/content/en/docs/user/kwok-manage-nodes-and-pods.md [hunk 13]
```diff
@@ -1,22 +1,22 @@
...
 
 ## Create a Node
 
-With Kwok, you can join arbitrary Node(s) just by simply creating `v1.Node` object(s):
+With kwok, you can join arbitrary Node(s) just by simply creating `v1.Node` object(s):
```
Comment (wzshiming) [inline site/content/en/docs/user/kwok-manage-nodes-and-pods.md:19]:
```suggestion
With `kwok`, you can join arbitrary Node(s) just by simply creating `v1.Node` object(s):
```

File: site/content/en/docs/user/kwok-manage-nodes-and-pods.md [hunk 14]
```diff
@@ -67,7 +67,7 @@ status:
 EOF
 ```
 
-After the Node is created, Kwok will maintain its heartbeat and keep it in the `ready` state.
+After the Node is created, kwok will maintain its heartbeat and keep it in the `ready` state.
```
Comment (wzshiming) [inline site/content/en/docs/user/kwok-manage-nodes-and-pods.md:70]:
```suggestion
After the Node is created, `kwok` will maintain its heartbeat and keep it in the `ready` state.
```

File: site/content/en/docs/user/kwok-manage-nodes-and-pods.md [hunk 15]
```diff
@@ -136,4 +136,4 @@ fake-pod-59bb47845f-wxn4b   1/1     Running   0          5s    10.0.0.1    kwok-
 
 ## Update spec of nodes or pods
 
-In a Kwok context, Nodes and Pods are nothing but pure API objects so feel free to mutate their API specs to do whatever simulation or testing you want.
+In a kwok context, Nodes and Pods are nothing but pure API objects so feel free to mutate their API specs to do whatever simulation or testing you want.
```
Comment (wzshiming) [inline site/content/en/docs/user/kwok-manage-nodes-and-pods.md:139]:
```suggestion
In a `kwok` context, Nodes and Pods are nothing but pure API objects so feel free to mutate their API specs to do whatever simulation or testing you want.
```

File: site/content/en/docs/user/stages-configuration.md [hunk 16]
```diff
@@ -8,7 +8,7 @@ This document walks you through how to configure the Stages of Lifecycle
 
 ## What is a Stage?
 
-The Stage API is a [Kwok Configuration](./configuration) that allows users to define and simulate different stages in the lifecycle of Kubernetes resources, such as nodes and pods.
+The Stage API is a [kwok Configuration](./configuration) that allows users to define and simulate different stages in the lifecycle of Kubernetes resources, such as nodes and pods.
```
Comment (wzshiming) [inline site/content/en/docs/user/stages-configuration.md:11]:
```suggestion
The Stage API is a [`kwok` Configuration](./configuration) that allows users to define and simulate different stages in the lifecycle of Kubernetes resources, such as nodes and pods.
```

File: site/content/en/docs/user/stages-configuration.md [hunk 17]
```diff
@@ -71,7 +71,7 @@ The `<expressions-string>` is provided by the [Go Implementation](https://github
 
 ### Node Stages
 
-This example shows how to configure the simplest and fastest stages of Node resource, which is also the default Node stages for Kwok.
+This example shows how to configure the simplest and fastest stages of Node resource, which is also the default Node stages for kwok.
```
Comment (wzshiming) [inline site/content/en/docs/user/stages-configuration.md:74]:
```suggestion
This example shows how to configure the simplest and fastest stages of Node resource, which is also the default Node stages for `kwok`.
```

File: site/content/en/docs/user/stages-configuration.md [hunk 18]
```diff
@@ -97,7 +97,7 @@ When applied, maintain the `status.conditions` field for the node.
 
 ### Pod Stages
 
-This example shows how to configure the simplest and fastest stages of Pod resource, which is also the default Pod stages for Kwok.
+This example shows how to configure the simplest and fastest stages of Pod resource, which is also the default Pod stages for kwok.
```
Comment (wzshiming) [inline site/content/en/docs/user/stages-configuration.md:100]:
```suggestion
This example shows how to configure the simplest and fastest stages of Pod resource, which is also the default Pod stages for `kwok`.
```

**Review by wzshiming — CHANGES_REQUESTED** [review]

File: site/content/en/menu/index.md (hunk 3 above)
Comment (windsonsea) [reply site/content/en/menu/index.md:10]: This snippet is added, not sure how to wrap it.

Comment (wzshiming) [inline site/content/en/menu/index.md:10]:
```suggestion
    - [`kwok` in Cluster]({{< relref "/docs/user/kwok-in-cluster" >}})
```

File: site/content/en/menu/index.md [hunk 19]
```diff
@@ -7,12 +7,12 @@ headless: true
...
+    - [kwok in Local]({{< relref "/docs/user/kwok-in-local" >}})
```
Comment (wzshiming) [inline site/content/en/menu/index.md:11]:
```suggestion
    - [`kwok` in Local]({{< relref "/docs/user/kwok-in-local" >}})
```

File: site/content/en/menu/index.md [hunk 20]
```diff
@@ -7,12 +7,12 @@ headless: true
...
+    - [kwokctl Manages Clusters]({{< relref "/docs/user/kwokctl-manage-cluster" >}})
```
Comment (wzshiming) [inline site/content/en/menu/index.md:12]:
```suggestion
    - [`kwokctl` Manages Clusters]({{< relref "/docs/user/kwokctl-manage-cluster" >}})
```

File: site/content/en/menu/index.md [hunk 21]
```diff
@@ -7,12 +7,12 @@ headless: true
...
     - [Manage Nodes and Pods]({{< relref "/docs/user/kwok-manage-nodes-and-pods" >}})
-  - [Kwokctl Snapshot Cluster]({{< relref "/docs/user/kwokctl-snapshot" >}})
-  - [Kwokctl Auditing]({{< relref "/docs/user/kwokctl-auditing" >}})
+  - [kwokctl Snapshots Cluster]({{< relref "/docs/user/kwokctl-snapshot" >}})
```
Comment (wzshiming) [inline site/content/en/menu/index.md:14]:
```suggestion
  - [`kwokctl` Snapshots Cluster]({{< relref "/docs/user/kwokctl-snapshot" >}})
```

File: site/content/en/menu/index.md [hunk 22]
```diff
@@ -7,12 +7,12 @@ headless: true
...
+  - [kwokctl Auditing]({{< relref "/docs/user/kwokctl-auditing" >}})
```
Comment (wzshiming) [inline site/content/en/menu/index.md:15]:
```suggestion
  - [`kwokctl` Auditing]({{< relref "/docs/user/kwokctl-auditing" >}})
```

**Pull request merged**